integrations/ivy/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "ivy",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --hmr",
+    "test": "ng build --prod",
+    "pretest": "npm install",
+    "postinstall": "cpx -v -C \"../../@ngxs/**/*\" \"node_modules/@ngxs/\""
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "9.0.0-next.3",
+    "@angular/common": "9.0.0-next.3",
+    "@angular/compiler": "9.0.0-next.3",
+    "@angular/core": "9.0.0-next.3",
+    "@angular/forms": "9.0.0-next.3",
+    "@angular/platform-browser": "9.0.0-next.3",
+    "@angular/platform-browser-dynamic": "9.0.0-next.3",
+    "@angular/router": "9.0.0-next.3",
+    "rxjs": "6.4.0",
+    "tslib": "1.10.0",
```

```suggestion
    "tslib": "file:../../node_modules/tslib",
```

And the rest packages expect of Angular. This will reduce installation time on the CI.